client/src/assets/icons/dumble.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+function DumbleIcon(
```

Comment:
`dumble` works, but I think you were going for `dumbbell` here? I'm fine leaving it, but feel free to update it - probably change the filename too if you do.